p2p_wallet/Scenes/Main/Sell/Pending/SellPendingCoordinator.swift

```diff
@@ -70,33 +70,36 @@ final class SellPendingCoordinator: Coordinator<SellPendingCoordinatorResult> {
             }
             .store(in: &subscriptions)
 
-        viewModel.send
-            .flatMap { [unowned self, navigationController] in
-                coordinate(to:
-                    SendCoordinator(
-                        rootViewController: navigationController,
-                        preChosenWallet: walletsRepository.nativeWallet,
-                        preChosenRecipient: Recipient(
-                            address: transaction.depositWallet,
-                            category: .solanaAddress,
-                            attributes: [.funds]
-                        ),
-                        preChosenAmount: transaction.baseCurrencyAmount,
-                        hideTabBar: true,
-                        source: .sell,
-                        allowSwitchingMainAmountType: false
+        if let nativeWallet = walletsRepository.nativeWallet {
+            viewModel.send
+                .flatMap { [unowned self, navigationController] in
+                    coordinate(to:
+                                SendCoordinator(
```

Comment:
Please fix indentation for better understanding the changes

Reply:
ok